src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="menu">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li>
+            <a
+              data-qa="hover"
+              class="nav__link"
+              href="#laptops-computers"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be empty lines between the <li> elements for better readability.